### PR TITLE
Kusto: Adding 2020-09-18 version

### PR DESCRIPTION
**Pull request by astauben**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Kusto/tools/Resources/.gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+* text=auto
```
Comment (dolauli) [inline src/Kusto/tools/Resources/.gitattributes:1]: The folder **tools** is not required to check-in. Please remove it.

**Review by dolauli — CHANGES_REQUESTED** [review]
Some changes are required.
Another potential issue is that I notice some cmdlets are changed, but there are no related changes in the **examples** folder.

File: src/Kusto/tools/Resources/.gitattributes (hunk 1 above)
Comment (astauben) [reply src/Kusto/tools/Resources/.gitattributes:1]: done

**Comment by astauben** [comment]
> Some changes are required.
> Another potential issue is that I notice some cmdlets are changed, but there are no related changes in the **examples** folder.



> Some changes are required.
> Another potential issue is that I notice some cmdlets are changed, but there are no related changes in the **examples** folder.

Hi, I've update the relevant example with the relevant new cmdlets

**Review by dolauli — CHANGES_REQUESTED** [review]
You should not remove the tools folder, which is not under src/Kusto.

**Comment by astauben** [comment]
> You should not remove the tools folder, which is not under src/Kusto.
@dolauli 
Done

**Comment by dolauli** [comment]
@astauben It looks recent swagger change breaks the tests of kusto. To avoid such kind of issue, I would suggest you lock the swagger through the commit id instead of using branch name. Please see https://github.com/Azure/azure-powershell/blob/generation/src/Databricks/readme.md for an example.

**Comment by astauben** [comment]
> @astauben It looks recent swagger change breaks the tests of kusto. To avoid such kind of issue, I would suggest you lock the swagger through the commit id instead of using branch name. Please see https://github.com/Azure/azure-powershell/blob/generation/src/Databricks/readme.md for an example.

Thanks, done

**Pull request merged**